apisupport/apisupport.ant/src/org/netbeans/modules/apisupport/project/universe/ModuleList.java

```diff
@@ -857,7 +857,7 @@ public boolean accept(File pathname) {
         static final String CNB = "org.netbeans.libs.junit4";
         JUnitPlaceholderEntry(File root) {
             super(CNB, new File(root, "platform/modules/" + CNB.replace('.', '-') + ".jar"),
-                    new File[] {new File(System.getProperty("user.home"), ".m2/repository/junit/junit/4.8.2/junit-4.8.2.jar")},
+                    new File[] {new File(System.getProperty("user.home"), ".m2/repository/junit/junit/4.8.2/junit-4.13.2.jar")},
```

Comment:
Shouldn't the parent directory's name (seems version of the artifact) change, too ?

Reply:
Fixed, thanks.